.gitignore

```diff
@@ -7,7 +7,7 @@
 #sbt
 /project/target/
 /project/project/target
-
+sonatype.sbt
```

Comment:
well, sbt release has some possibly per project options, so I put it here, but it won't publish a dirty directory.